app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -130,6 +130,16 @@ def link
         end
       end
 
+      def page_for_link
+        @page = Page.find(params[:id])
+        @area_name = params[:area_name]
+        @url_prefix = ''
+        render partial: 'page_for_links',
+          object: @page,
+          locals: {area_name: @area_name},
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.